statsmodels/regression/tests/test_lme.py

```diff
@@ -86,6 +88,8 @@ def f(x):
 class TestMixedLM(object):
 
     # Test analytic scores and Hessian using numeric differentiation
+    @pytest.mark.skipif(PLATFORM_OSX, reason='known to fail on OSX due to '
+                                             'numerical differences')
```

Comment:
shouldnt this be an xfail?  the test is failing for poorly-understood reasons, not invalid on OSX.